### ISSUE TITLE
Task run_tests_managed no longer available.

### ISSUE THREAD

**Issue by swagattalsania** [SalesforceFoundation/CumulusCI#657]
Hello team,
I just noticed that the task run_tests_managed is no longer available with the latest version of CumulusCI. We have a CI job that upgrades all the dependencies before running the tasks. Upgrading CumulusCI is one of the tasks in it. This broke our existing job of running the test methods of our managed package. I had to change my job to use run_tests task which is fine.

What I am worried about is, will CumulusCI team be making such breaking changes in the future as well? Isn't it supposed to be backward compatible so that it does not break any existing implementations that the users might have?

**Comment by jlantz** [comment]
This was the last change we needed to make which we knew would be disruptive to existing customizations.  We tried to call this out in the release notes: https://github.com/SalesforceFoundation/CumulusCI/blob/master/HISTORY.rst#200-beta98-2018-05-31

We made this change because it allowed us to take what we've learned from building per project customizations of CumulusCI over the last 17 months and incorporate the best practices into CumulusCI.  We also added support for flows to call other flows which opened the possibility of restructuring the default flows.

Our intent is that this was a one time breaking change.  We removed the beta label after making this change.  CumulusCI is now officially out of beta and we wanted to make sure we made those breaking changes before doing so.  Sorry for the confusion.
